eq-author-api/schema/tests/comments.test.js

```diff
@@ -152,6 +151,22 @@ describe("comments", () => {
     expect(comment.comments).toHaveLength(0);
   });
 
+  describe("locking", () => {
+    it("should prevent adding comments to a locked questionnaire", async () => {
+      expect(
+        createComment(
+          { ...ctx, questionnaire: { ...ctx.questionnaire, locked: true } },
+          {
+            componentId,
+            commentText: "a new comment is created",
+          }
+        )
+      ).rejects.toThrow();
+
+      questionnaire.locked = false;
+    });
```

Comment:
You could enhance this test by checking that the type of error thrown and message it's thrown with is what is expected.